### PR TITLE
Inventory Advisor cleanups n changes 👨‍🚒🍇🌌

### PR DESCRIPTION
**Pull request by AllenBW**
fixes https://projects.engineering.redhat.com/browse/RHCLOUD-5873

#### looks like this
Screen Shot 2020-04-14 at 11 24 11 AM
Screen Shot 2020-04-14 at 11 23 53 AM

### CODE REVIEW

File: packages/inventory-insights/src/Api.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import axios from 'axios';
+
+export default {
```
Comment (AllenBW) [inline packages/inventory-insights/src/Api.js:3]: no we don't use all of these, but 🤷 doesn't hurt to have em around

Comment (karelhala) [reply packages/inventory-insights/src/Api.js:3]: You should consider using [@redhat-cloud-services/insights-client](https://www.npmjs.com/package/@redhat-cloud-services/insights-client) at some point. No need to call URLs directly.

File: packages/inventory-insights/src/Insights.js [hunk 2]
```diff
@@ -404,7 +404,7 @@ class InventoryRuleList extends Component {
                     actionsConfig={{ actions }}
                     bulkSelect={bulkSelect}
                     filterConfig={{ items: filterConfigItems }}
-                    pagination={<React.Fragment> {results === 1 ? `${results} recommendation` : `${results} recommendations`} </React.Fragment>}
+                    pagination={<React.Fragment> {results === 1 ? `${results} recommendation` : `${results} Recommendations`} </React.Fragment>}
```
Comment (karelhala) [inline packages/inventory-insights/src/Insights.js:407]: Plural has capital R and singular has lower r, shouldn't this be the same?

Comment (AllenBW) [reply packages/inventory-insights/src/Insights.js:407]: yup good call

**Comment by AllenBW** [comment]
OOOO DIDN'T KNOW ABOUT @redhat-cloud-services/insights-client thats hella neat 🤔 not here... maybe again in da... futureeeeeee 🤗

**Pull request merged**